domain/src/test/java/org/oppia/domain/topic/TopicListControllerTest.kt

```diff
@@ -38,17 +38,17 @@ import org.oppia.app.model.ProfileId
 import org.oppia.app.model.PromotedStory
 import org.oppia.util.caching.CacheAssetsLocally
 import org.oppia.util.data.AsyncResult
+import org.oppia.util.gcsresource.DefaultResourceBucketName
 import org.oppia.util.logging.EnableConsoleLog
 import org.oppia.util.logging.EnableFileLog
 import org.oppia.util.logging.GlobalLogLevel
 import org.oppia.util.logging.LogLevel
 import org.oppia.util.parser.DefaultGcsPrefix
-import org.oppia.util.parser.DefaultGcsResource
 import org.oppia.util.parser.ImageDownloadUrlTemplate
 import org.oppia.util.threading.BackgroundDispatcher
 import org.oppia.util.threading.BlockingDispatcher
 import org.robolectric.annotation.Config
-import java.util.Date
+import java.util.*
```

Comment:
Remove wildcard import.

Reply:
+1

Reply:
Done. It's unfortunate that Android Studio sometimes defaults to wildcards despite being configured to never use them. :?

Reply:
Per time constraints, marking this as done. Please let me know if you disagree.